fineract-doc/src/docs/en/chapters/documentation/transaction-idempotency.adoc

```diff
@@ -0,0 +1,118 @@
+= Introducing Transaction Idempotency into Fineract
+
+For banks, Apache Fineract is introducing transaction idempotency, business and
+effective date configuration as an addition to currenty supported
+
+* System date
```

Comment:
Is there a reason to mixing up the idempotency concept with dates? I don't get it.